IdentityCore/src/cache/accessor/MSIDLegacyTokenCacheAccessor.m

```diff
@@ -179,24 +179,43 @@ - (MSIDRefreshToken *)getRefreshTokenWithAccount:(MSIDAccountIdentifier *)accoun
                                          context:(id<MSIDRequestContext>)context
                                            error:(NSError **)error
 {
-    MSID_LOG_VERBOSE(context, @"(Legacy accessor) Get refresh token with authority %@, clientId %@, familyID %@", configuration.authority, configuration.clientId, familyId);
-    MSID_LOG_VERBOSE_PII(context, @"(Legacy accessor) Get refresh token with authority %@, clientId %@, familyID %@, account %@", configuration.authority, configuration.clientId, familyId, account.homeAccountId);
+    return [self getRefreshableTokenWithAccount:account
+                                       familyId:familyId
+                                 credentialType:MSIDRefreshTokenType
+                                  configuration:configuration
+                                        context:context
+                                          error:error];
+}
+
+- (MSIDRefreshToken *)getRefreshableTokenWithAccount:(MSIDAccountIdentifier *)account
```

Comment:
nit: this and the calling method, why is the type AccountIdentifier and we call it an account?

Reply:
agree, fixed :)